doc/developer-notes.md

```diff
@@ -83,7 +83,7 @@ code.
     separate words (snake_case).
     - Class member variables have a `m_` prefix.
     - Global variables have a `g_` prefix.
-  - Compile-time constant names are all uppercase, and use `_` to separate words.
+  - Constant names are all uppercase, and use `_` to separate words.
```

Comment:
There's a name for that :)

`- Constant names are written in screaming snake case (e.g. all uppercase and using _ to separate words).`

Reply:
aka SCREAMING_SNAKE_CASE :)

Reply:
or just

`- Constants are written in SCREAMING_SNAKE_CASE.`